cmd/skywire-cli/commands/hv/hvdmsg/hvdmsg.go

```diff
@@ -0,0 +1,90 @@
+package hvdmsg
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+	"github.com/toqueteos/webbrowser"
+
+	"github.com/skycoin/skywire/pkg/visor/visorconfig"
+)
+
+var (
+	path string
+	pkg  bool
+	pk   string
+	url  string
+)
+
+func init() {
+	RootCmd.AddCommand(dmsgUICmd)
+	dmsgUICmd.Flags().StringVarP(&path, "input", "i", "", "read from specified config file")
+	dmsgUICmd.Flags().BoolVarP(&pkg, "pkg", "p", false, "read from "+visorconfig.Pkgpath)
+	dmsgUICmd.Flags().StringVarP(&pk, "visor", "v", "", "public key of visor to connect to")
+
+	RootCmd.AddCommand(dmsgURLCmd)
+	dmsgURLCmd.Flags().StringVarP(&path, "input", "i", "", "read from specified config file")
+	dmsgURLCmd.Flags().BoolVarP(&pkg, "pkg", "p", false, "read from "+visorconfig.Pkgpath)
+	dmsgURLCmd.Flags().StringVarP(&pk, "visor", "v", "", "public key of visor to connect to")
+}
+
+var dmsgUICmd = &cobra.Command{
+	Use:   "ui",
+	Short: "Open dmsgpty UI in default browser",
+	Run: func(_ *cobra.Command, _ []string) {
+		if pk == "" {
+			if pkg {
+				path = visorconfig.Pkgpath
+			}
+			if path != "" {
+				conf, err := visorconfig.ReadFile(path)
+				if err != nil {
+					log.Fatal("Failed:", err)
+				}
+				url = fmt.Sprintf("http://127.0.0.1:8000/pty/%s", conf.PK.Hex())
+			} else {
+				client := rpcClient()
+				overview, err := client.Overview()
+				if err != nil {
+					log.Fatal("Failed to connect:", err)
```

Comment:
Maybe leave a message here to ask the user if the visor is running...Applies for some other commands as well.

Reply:
noted

Reply:
fixed more than just this instance with next commit